examples/flask/app.py

```
@@ -0,0 +1,34 @@
+def home():
+    return '<a href="/login">Login</a> | <a href="/account">Account</a> | <a href="/logout">Logout</a>'
+
+
+@app.route("/account")
+@auth.requires_auth
+def account():
+    return auth.current_user
+
+
+if __name__ == "__main__":
+    app.run(port=int(os.environ.get("PORT", 5001)), debug=True)
```

Ruff (0.16.1)

[error] Use of `debug=True` in Flask app detected

(S201)